node_modules/oae-core/documentpreview/js/documentpreview.js

```diff
@@ -108,14 +108,14 @@ define(['jquery', 'underscore', 'oae.core', 'lazyload'], function($, _, oae) {
             page.height = page.$el.height();
             page.marginTop = CONTENT_PAGE_SPACING;
 
-            // Request the page content
+            // Request the page content HTML version
             $.ajax({
```

Comment:
Having them combined would be a lot cleaner from a UI point of view and wouldn't require as many individual requests. Having said that, I see your point about storage backends, so we should probably go for the separate requests approach here.